src/client/app/containers/CompareChartContainer.js

```diff
@@ -145,8 +151,8 @@ function mapStateToProps(state, ownProps) {
 		},
 		title: {
 			display: true,
-			text: delta((-1 + (((currentWeek / currentLastWeek) * lastWeek) / lastWeek))),
-			fontColor: colorize((-1 + (((currentWeek / currentLastWeek) * lastWeek) / lastWeek)))
+			text: delta((-1 + (((current / currentPrev) * prev) / prev))),
```

Comment:
Recommend sticking `(currentWeek / currentLastWeek) * lastWeek) / lastWeek)` in a variable here.